.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+<!--
+General questions should be asked on the mailing list modin-dev@googlegroups.com.
+
+Before submitting an issue, please fill out the following form.
+-->
+
+### System information
+- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
+- **Modin installed from (source or binary)**:
+- **Modin version**:
+- **Python version**:
+- **Exact command to reproduce**:
+
+<!--
+You can obtain the Modin version with
+
+python -c "import modin; print(modin.__version__)"
```

Comment:
This doesn't work yet. We will need to make sure that we add this right before the release.

Reply:
This works as of #24